website/docs/en/guide/tech/solid.mdx

```diff
@@ -2,14 +2,14 @@
 
 ## How to Use
 
-Rspack provides two solutions to support Solid:
+Rspack provides two solutions to support SolidJS:
```

Comment:
We can just call `SolidJS` as `Solid`:

![image](https://github.com/web-infra-dev/rspack/assets/7237365/8cc1f7dc-4781-47f2-8449-5779e8bacff5)

https://www.solidjs.com/guides/getting-started